lib/TH/THLapack.h

```diff
@@ -3,15 +3,21 @@
 
 #include "THGeneral.h"
 
+#define THErrorHandler(X) X
```

Comment:
I have since found further leaks elsewhere, for which I need the same construction, so I propose instead to

     #define THCleanup(...) __VA_ARGS__

in `THGeneral.h`. We will then reuse that pattern both in TH and in THLapack.